tests/suites/0_stateless/02_0041_function_strings_format.sql

```diff
@@ -0,0 +1,14 @@
+SELECT FORMAT(12332.123456, 4);
+SELECT FORMAT(-12332.123456, 4);
+SELECT FORMAT(12332.1, 4);
+SELECT FORMAT(12332.2, 0);
+SELECT FORMAT(12332.2, -1);
+SELECT FORMAT(12332, 2);
```

Comment:
What if `select format(number +10000, 2)  from numbers(10)`?

Reply:
It is ok when `select format(number +10000, 2) from numbers(1)` but error when `numbers(2)`.

Reply:
It is ok when `select format(number +10000, 2) from numbers(1)` but error when `numbers(2)`.

Reply:
As I mentioned before, we should care about none constant case.

See https://github.com/datafuselabs/databend/blob/f5fda2e8f7c13f2c4764d941472fe7801693381d/common/functions/src/scalars/strings/repeat.rs

Reply:
Copy from repeat() and it works!